test.py

```diff
@@ -40,665 +41,601 @@
+				"Un sentiero che sembra finire in una foresta"
+			],
+			"exits": {
+				"Canyon": [
+					"verso le rocce di un canyon",
+					"verso un canyon di terra rossa"
+				],
+				"Lago-outer": [
+					"verso la riva di un lago lucente",
+					"verso uno specchio d'acqua"
+				],
+				"Foresta-inner": [
+					"al centro della foresta",
+					"in una parte di foresta ancora più fitta"
+				]
+			},
+			"variants": {}
+		},
+
+		"Foresta-inner": {
+			"chances": 5,
+			"descriptions": [
+				"Una foresta fittissima",
+				"Una foresta che sembra aver ospitato un'antica civiltà"
+			],
+			"exits": {
+				"Grotta-outer": [
+					"verso una grotta",
+					"verso quella che sembra una miniera abbandonata"
+				],
+				"Foresta-inner": [
+					"dove gli alberi sembrano meno fitti",
+					"dove la foresta sembra diradarsi"
+				],
+				"Monastero-outer": [
+					"all'entrata di un monastero abbandonato",
+					"davanti alla porta di un palazzo antico"
+				],
+				"Foresta-outer": [
+					"verso dove gli alberi sembrano diradarsi",
+					"verso un sentierino"
+				]
+			},
+			"variants": {
+				"more": {
+					"contained": ObjectGroup([Food(Description('mora', 'more', 'mora', 'more', '','una', 'la', 'le', 'delle'), nutrition=5)]*5)
+				},
+
+				"albero": {
+					"contained": ObjectGroup([Object(Description('albero maestoso', 'alberi maestosi', 'albero', 'alberi', 'che dalla dimensione ha probabilmente centinaia di anni','un', 'l\'', 'gli', 'degli'), visible=True)])
+				},
+
+				"acqua": {
+					"actions": [Action('Bevi',
+						EventMessages(
+							subject='bevi un po\' d\'acqua',
+							close="{subject} beve un po' d'acqua"),
+						lambda player: setattr(player, 'thirst', player.thirst-40),
+						2.5, 0,)
+					]
+				},
+			}
+		},
+
+		"Monastero-outer": {
+			"chances": 5,
+			"descriptions": [
+				"L'entrata di un antico monastero",
+				"Una porta di quello che sembra un monastero abbandonato"
+			],
+			"exits": {
+				"Montagna-outer": [
+					"verso il sentiero da cui sei arrivato",
+					"al sentiero di prima"
+				],
+				"Foresta-inner": [
+					"verso la foresta",
+					"alla foresta"
+				],
+				"Monastero-inner-1": [
+					"dentro il monastero",
+					"all'interno del palazzo"
+				]
+			},
+			"variants": {}
+		},
+
+		"Monastero-inner-1": {
+			"chances": 5,
+			"descriptions": [
+				"Il piano terra del monastero. Non sembra esserci nessuno",
+				"Un atrio con due rampe di scale"
+			],
+			"exits": {
+				"Monastero-outer": [
+					"all'aperto",
+					"all'esterno del monastero"
+				],
+				"Monastero-inner-2": [
+					"verso il piano di sopra",
+					"di sopra"
+				]
+			},
+			"variants": {}
+		},
+
+		"Monastero-inner-2": {
+			"chances": 5,
+			"descriptions": [
+				"Il primo piano del monastero. Si vedono dei letti",
+				"Probabilmente la vecchia mensa del monastero"
+			],
+			"exits": {
+				"Monastero-inner-1": [
+					"da dove sei arrivato",
```

Comment:
queste non vanno bene, il giocatore potrebbe addirittura spawnare qui, e "torna" è aggiunto in automatico. Fai semplicemente verso le scale che scendono